docs/management_api.md

```diff
@@ -13,6 +13,7 @@ The Management API listens on port 8081 and is only accessible from localhost by
 
 Similar to the [Inference API](inference_api.md), the Management API provides a [API description](#api-description) to describe management APIs with the OpenAPI 3.0 specification.
 
+Alternatively, if you want to use KServe, TorchServe supports both v1 and v2 API. For more details please look into this [kserve/README.md](https://github.com/pytorch/serve/tree/master/kubernetes/kserve)
```

Comment:
`kserve/README.md` as the hyperlink text can be confusing as someone might try to look up a kserve directory which don't have.

```suggestion
Alternatively, if you want to use KServe, TorchServe supports both v1 and v2 API. For more details please look into this [kserve documentation](https://github.com/pytorch/serve/tree/master/kubernetes/kserve)
```